frontend/src/__tests__/__snapshots__/Worksheet.test.js.snap

```diff
@@ -35,7 +35,7 @@ Object {
                       class="editable-field"
                       style="color: rgb(34, 94, 168);"
                     >
-                      &lt;none&gt;
+                      &lt;enter&gt;
```

Comment:
Pull this out into a separate function...  Not sure '<enter>' is more intuitive than '<none>'...

Either way, maybe we can italicize?

Reply:
> Pull this out into a separate function... Not sure '' is more intuitive than ''...

@percyliang can you clarify what you mean by this?